### ISSUE TITLE
Testing Bug - Accessibility Tag

### ISSUE THREAD

**Issue by sarahlazarich** [bloom-housing/bloom#4889]
Steps to Reproduce 
- Add ANY accessibility feature to a listing in https://bloom-detroit-partners.netlify.app/
- View that listing https://bloom-detroit.netlify.app/

Expected Behavior 
- This should display an "accessible" tag ([Example](https://detroit-public-dev.netlify.app/listing/4b689eed-4d69-433b-865d-61864781e7cb/every_field_filled_out_434_bennington_ct_saline_mi)) [Here is the new design using the warn color and a new icon](https://www.figma.com/design/xqyamlgdsASkLTYlcifBwq/core-patterns?node-id=2370-4788&m=dev)
<img width="495" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNjh0Q2c9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--69f896f72bbaec32f9d87a536663f7ec87c02e12/image.png" alt="image.png" />
 

Current Behavior
- This does not display any tag

**Comment by ColinBuyck** [comment]
@matzduniuk Update on this after talking with @sarahlazarich. In my review, I was making two assumptions (that I imagine you were making too) when I approved the PR completing this work. The first being that these new tags should only exist on the listing card shown in the under construction view and two being that all of the new [Accessibility tags should be included](https://www.figma.com/design/xqyamlgdsASkLTYlcifBwq/core-patterns?node-id=2370-4788&m=dev) rather than only the Accessible tag.

Correct anything here (@sarahlazarich)
1) only the "Accessible" tag should be added and the "Hearing Units", "Visual Units", etc. should not be included in this work despite being in the design. None of the gray-colored tags should be added)
2) The accessible tag should show on all public representations of the listing. The listing card on the homepage, the listing card on the listings page, and on the listings detail view. As a model, check out the current implementation of Detroit to see these examples. https://homeconnect.detroitmi.gov/

Apologies for the confusion and the premature approval
